.github/main.workflow

```diff
@@ -0,0 +1,11 @@
+workflow "Test ghpages" {
+  on = "push"
```

Comment:
Need to use the correct event trigger. This is just for testing
https://developer.github.com/actions/creating-workflows/workflow-configuration-options/#actions-attributes